### PR TITLE
SW-2583 / SW-2574 Show planting site data for sites that don't have zones/plots

### PR DESCRIPTION
**Pull request by karthikbtf**
- code was only processing zones/plots driven info
- using a separate query now for totals and by species
- also show map on mobile for planting sites with plots even if there are no plants
- also show the plot selector for empty state in species chart

Terraware App 2022-12-21 09-19-00

### CODE REVIEW

File: src/components/Plants/PlantingSiteDetails.tsx [hunk 1]
```diff
@@ -27,6 +26,11 @@ export type PlantingSiteZone = {
   plots: PlantingSitePlot[];
 };
 
+export type PlantingSiteSpecies = {
```
Comment (constanzauanini) [inline src/components/Plants/PlantingSiteDetails.tsx:29]: This has the same attributes than Population, could we use that type?

Comment (karthikbtf) [reply src/components/Plants/PlantingSiteDetails.tsx:29]: good catch 👍

Comment (karthikbtf) [reply src/components/Plants/PlantingSiteDetails.tsx]: fixed

**Comment by karthikbtf** [comment]
@constanzauanini i also added fix for SW-2574
- also show map on mobile for planting sites with plots even if there are no plants
- also show the plot selector for empty state in species chart

**Review by constanzauanini — APPROVED** [review]
Looks good! Tested mobile on vercel

**Comment by karthikbtf** [comment]
> Looks good! Tested mobile on vercel

Thank you 🙏

**Pull request merged**